hexa/workspaces/models.py

```diff
@@ -30,6 +38,7 @@ def create_if_has_perm(
             raise PermissionDenied
 
         create_kwargs = {"name": name, "description": description}
+        create_kwargs["slug"] = create_workspace_slug(name)
```

Comment:
Can we be more strict with slug creation and validation ? Make sure we only have lowercase letters, numbers and dashes?

Reply:
From my understanding of slugify, that's exactly what it does. It's makes the text lowercase, replace spaces by dashes ... It is enough no ?

Reply:
It also accepts underscores, which we don't want (not valid for bucket / db names)

Reply:
Regarding `slugify`, by default it does not.

```python
from slugify import slugify
slugify('a_string_with_underscores_')
'a-string-with-underscores'
```

The `validate_slug` of Django does accept it, I'll make a custom validator.

Reply:
Done